### DISCUSSION TITLE
[BUG] White background behind transparent elements

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#53] [General]
Hi,

I want to discuss this also here:

they is a nasty layer-issue.
When using layers, groups, element (also when using them as symbols) they seem to generate a white background.
Also the Typo seems to be a little bit squeezed.

How to fix this issue?

<img width="991" alt="131871959-9a64d743-dabb-42b2-b876-bb1f8057c19e" src="https://user-images.githubusercontent.com/2456068/133601360-634677ad-a985-4aef-bbea-a7a4a17de601.png">

**Comment by MaxBazarov** [comment]
Never seen this issue before.
Could you please share a sample file?

**Comment by Nevu** [comment]
Have a look at this. Sketch + Puzzle

[layer-issues.zip](https://github.com/ingrammicro/puzzle-publisher/files/7178344/layer-issues.zip)

**Reply by MaxBazarov** [reply]
I see. The problem is generating by a conjunction of two things:
1) You have selected JPG as an export format.
2) You marked the text layer as "Fixed position when scrolling".

But JPG format doesn't support a transparency, so Puzzle Publisher can't show fixed layers correctly. I think the best solution will be switch fixed layers to PNG automatically. 

Right now you can select PNG for exporting.

**Reply by Nevu** [reply]
Thank you. I try out!
